### ISSUE TITLE
Issues with 4.2

### ISSUE THREAD

**Issue by elinw** [ropensci/skimr#702]
So it looks like the switch from a warning to error  for `if()` of objects of length 1 revealed that we had a hidden warning.  This becoming an error then led to cascading other problems.
For reasons I'm not sure of there was also a problem with finding reassign_skim_attributes. 
All this is why the original issue of pkgdown not building arose.

**Comment by michaelquinn32** [comment]
I just checked this against the current develop branch (after merging in all changes for the next version)

```
devtools::check(env_vars = c("_R_CHECK_LENGTH_1_CONDITION_" = "true"))
```

Once the fix for #700 is in, can you let me know if this is still an issue? Thanks!

**Comment by elinw** [comment]
I think we're good!
